analyzers/tests/SonarAnalyzer.UnitTest/TestCases/LinkedListPropertiesInsteadOfMethods.vb

```diff
@@ -0,0 +1,25 @@
+﻿Imports System.Collections
+Imports System
+Imports System.Collections.Generic
+Imports System.Linq
+
+Class [MyClass]
+    Private Sub Various()
+        Dim data = New LinkedList(Of Integer)()
+        Enumerable.First(data) ' Noncompliant {{'First' property of 'LinkedList' should be used instead of the 'First()' extension method.}}
```

Comment:
This rule is actually a bit different for VB. `data.First()` is calling the readonly property of the LinkedList (and not the Enumerable method). This is why VB has a slightly different implementation in the first place because in order to "trigger the rule" you must be explicit and call `Enumerable.First(data)`. In other words, `data.First()` and `data.First.Value` are both compliant. Does it make sense? (small snippet [here](https://sharplab.io/#v2:EYLgtghglgdgNAGxAN2HALiCBndcAmIA1AD4CSYADgPYBO62ABAMoCeuApmAHQDC1CBBwDG6KNRjZuAcQ4wOtKMICwAKAo16TNpx4AZWAEc1a3ghxMA2gFlWZiwF01jF4wAKi5BHQcWAV2BGADUIRWo/bAAKAEpnV3iAESgwRnxvCEYAXkYAOQ4Ad0YDGABrDnwDXEiAeQAzRjIYHwBzBWiYuPiXAFEYPzAFCGAhbgAxKFoqtPQIaK74gHJciWFqKgQoCCbOrqSUjOzpiDGJqrn5xcZ+dc3t1QvGPcZAw/STyfRuEIQ/X3ml66UDZbdA7Xr4fzANTgq7mbDYNRAA))